chaincode/tsconfig.json

```
@@ -7,7 +7,7 @@
     "composite": true
   },
   "files": [],
-  "include": ["**/*", "package.json"],
+  "include": ["**/*", "package.json", "../chain-api/src/types/DexUserPositions.ts"],
```

```suggestion
  "include": ["**/*", "package.json"],
```

This broke the CI/CD build, as we see here: https://github.com/GalaChain/sdk/actions/runs/13461741671/job/37618510302?pr=508

Please work with the existing monorepo setup / nx, do not try to include relative paths to other modules.